<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditServeurMonoWidget</class>
 <widget class="QWidget" name="AuditServeurMonoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AuditServeurMonoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Num√©ro de port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPort">
       <property name="minimum">
        <number>1025</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonLancemenServeur">
     <property name="text">
      <string>Lancement serveur</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelInformations">
     <property name="text">
      <string>Informations</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditInformations"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonQuitter">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditServeurMonoWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>726</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
